### ISSUE TITLE
Recommend adding a logging level to ojph_wrapper.cpp

### ISSUE THREAD

**Issue by palemieux** [aous72/OpenJPH#146]
This allows an application to disable info level messages, which otherwise fill the console, e.g., if codestreams are truncated.

```
  ////////////////////////////////////////////////////////////////////////////
  EMSCRIPTEN_KEEPALIVE
  void set_message_level(int level)
  {
    if (ojph::INFO >= level && level != ojph::NO_MSG) {
      ojph::set_info_stream(stdout);
    } else {
      ojph::set_info_stream(NULL);
    }

    if (ojph::WARN >= level && level != ojph::NO_MSG) {
      ojph::set_warning_stream(stdout);
    } else {
      ojph::set_warning_stream(NULL);
    }

    if (ojph::ERROR >= level && level != ojph::NO_MSG) {
      ojph::set_error_stream(stdout);
    } else {
      ojph::set_error_stream(NULL);
    }
  }
```

**Comment by aous72** [comment]
Hi Pierre,

Thank you for suggesting this.

This can be part of your code.  
My thinking is that it is sufficient for me to provide `set_XXXX_stream` for the end user to choose where things go.
A second option for customization is to define your own class by deriving from `class message_XXX` to do whatever you like responding to errors/warning/info; then, using `configure_XXXX' to start using this new class it. 

If the code is inside `ojph` namespace, a better version of the code might be 
```
void set_message_level(int level)
{
  if (OJPH_MSG_LEVEL::ERROR > level)
    ojph::set_error_stream(NULL);

  if (OJPH_MSG_LEVEL::WARN > level) 
    ojph::set_warning_stream(NULL);

  if (OJPH_MSG_LEVEL::INFO > level)
    ojph::set_info_stream(NULL);
}
```
There is also the possibility of
```void set_message_level(OJPH_MSG_LEVEL level)```

Let me think about it.

Kind regards,
Aous.

**Comment by aous72** [comment]
This issue was addressed in release 0.15.0.
If there are any issues, please reopen this issue.
